Tools/i18n/pygettext.py

```diff
@@ -5,7 +5,7 @@
 # Minimally patched to make it even more xgettext compatible
 # by Peter Funk <pf@artcom-gmbh.de>
 #
-# 2002-11-22 J�rgen Hermann <jh@web.de>
+# 2002-11-22 Jürgen Hermann <jh@web.de>
```

Comment:
What's going on here?

Reply:
Oh strange, github displays this with a very weird diff on the "files changed" tab, but doesn't in the "conversation" tab

Reply:
I'm not sure what to make of this really. this and the other line you commented on below were added to my commit without me even changing them. I then tried to undo the change (see the 2nd and 3rd commits) to no-avail. After copying and pasting the char in and committing it seems we're where we started, but GH still shows the line as replaced. This suggests to me that perhaps the encoding has changed. :shrug:

Reply:
Ah yes, there's a very funky encoding specified for this file on line 2. I seem to remember something about github having some weird behaviour around this kind of thing in the past.

I'll see if I can fix it...

Reply:
I fixed it by running this sequence of commands, for future reference:

```
> git checkout main -- Tools/i18n/pygettext.py
> git restore --staged Tools
> git restore -p Tools
```

The `-p` option with `git restore` allows you to interactively choose which "hunks" in the diff to keep, and which to discard :)